### ISSUE TITLE
Chinese (Simplified) translation lost

### ISSUE THREAD

**Issue by marsxxl** [microsoft/Localization#71]
VS Code Version 1.36.0 (1.36.0) 
Chinese (Simplified) Version 1.36.2
**“按键映射”**
![](https://i.imgur.com/fiIT1RR.png)

**Comment by aldonetti** [comment]
Thank you @xinxiaoliang we are looking into it

**Comment by yanshouwang** [comment]
And it's right in the menu, but wrong in the setting icon's popup
![image](https://user-images.githubusercontent.com/24455689/60776046-99722000-a15b-11e9-9efd-c7e32076911b.png)
![image](https://user-images.githubusercontent.com/24455689/60776050-a989ff80-a15b-11e9-89f8-77b3a9822a7a.png)

**Comment by dbaeumer** [comment]
The strings seems to be correctly localize in  VS Code source code. The locations are:

https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/contrib/extensions/electron-browser/extensionsActions.ts#L1829

https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/contrib/extensions/electron-browser/extensions.contribution.ts#L299

https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/contrib/extensions/electron-browser/extensions.contribution.ts#L308

**Comment by weeteckt** [comment]
Thanks for the feedback and this will be fixed in the next language pack update.

@janbaltus to implement this fix.

**Comment by barnabasn** [comment]
This is now fixed in the last (1.37.2) published Language Pack:
![image](https://user-images.githubusercontent.com/38212935/62300345-e6a19300-b476-11e9-91a8-266b5ab9a243.png)

**Comment by yanshouwang** [comment]
But it's not fixed with mine, I am alreay update to the latest pack @barnabasn 
![image](https://user-images.githubusercontent.com/24455689/62309888-efb45380-b4ba-11e9-9d19-1adb0fb66ec3.png)

**Comment by yanshouwang** [comment]
And this "Open recent" also lost, what happened for me?
![image](https://user-images.githubusercontent.com/24455689/62310104-5df91600-b4bb-11e9-9e3d-93791c2ebb47.png)

**Comment by bulletcoder** [comment]
The 1.37.2 zh-cn language pack causes more severe problems, as all the setting menus in extensions lost the translation. (testing in windows and mac os)
<img width="481" alt="屏幕快照 2019-08-02 02 36 22" src="https://user-images.githubusercontent.com/38938372/62311126-70c91600-b4ce-11e9-83e3-d1ef0e44ac68.png">

**Comment by bulletcoder** [comment]
> The 1.37.2 zh-cn language pack causes more severe problems, as all the setting menus in extensions lost the translation. (testing in windows and mac os)
> <img alt="屏幕快照 2019-08-02 02 36 22" width="481" src="https://user-images.githubusercontent.com/38938372/62311126-70c91600-b4ce-11e9-83e3-d1ef0e44ac68.png">

I hope the admin can revert the 1.37.2 update asap.

**Comment by bulletcoder** [comment]
> The 1.37.2 zh-cn language pack causes more severe problems, as all the setting menus in extensions lost the translation. (testing in windows and mac os)
> <img alt="屏幕快照 2019-08-02 02 36 22" width="481" src="https://user-images.githubusercontent.com/38938372/62311126-70c91600-b4ce-11e9-83e3-d1ef0e44ac68.png">

and the previous 1.37.1 did not fix "Keymap" translation in the bottom left setting menu.

**Comment by weeteckt** [comment]
we are looking into this.

**Comment by barnabasn** [comment]
The fix for the original issue ("Keymaps") was meant only for VS Code version 1.37.0. or above (e.g. for VS Code Insiders).
I reverted (republished) the previous version of Language Pack for 1.36 with LP version 1.37.3.
Plus, published the latest Language Pack (with fixed translation of "Keymaps") for engine 1.37.0 or above with LP version 1.37.4.
I tested the result with both the standard Visual Studio Code and Insiders versions. The missing translation of "Keymaps" is fixed in the VS Code Insiders and rest of strings are localized in the standard VS Code as well.
